README.md

```diff
@@ -1,72 +1,72 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# Project Name
+## Capastone Project for the Html,CSS,Javascipt module
+
+This project is to help demostrating or put into practice what we have learned at Microverse for the time we spent at Microverse program until now. Please have a look at my Live site and give me feedback Live Demo. 🙂
+
+- Destop Version
+
+![Screen Shot 2022-03-10 at 13 50 42](https://user-images.githubusercontent.com/38294716/157656384-6cd10034-057e-456d-8c34-8cc3c780a268.png)
+![Screen Shot 2022-03-10 at 13 50 54](https://user-images.githubusercontent.com/38294716/157656407-a52f1ebf-9de0-46a6-ab90-90c2e8845ba9.png)
+![Screen Shot 2022-03-10 at 13 51 05](https://user-images.githubusercontent.com/38294716/157656416-f14619de-a8b9-4063-8331-748140df5322.png)
+![Screen Shot 2022-03-10 at 13 51 15](https://user-images.githubusercontent.com/38294716/157656436-90224d3b-440a-427e-b0a3-64ab0f672ca8.png)
+
+- Mobile Version
+
+![Screen Shot 2022-03-10 at 13 53 59](https://user-images.githubusercontent.com/38294716/157656834-4b4f657b-84b3-4a4c-ba89-552d1f020b0b.png)
+![Screen Shot 2022-03-10 at 13 54 17](https://user-images.githubusercontent.com/38294716/157656844-25e7e0d2-488c-4be9-a013-95622deaf52a.png)
+![Screen Shot 2022-03-10 at 13 54 26](https://user-images.githubusercontent.com/38294716/157656855-73f792f7-2e78-4b7e-8d6e-4e883236881e.png)
+![Screen Shot 2022-03-10 at 13 54 34](https://user-images.githubusercontent.com/38294716/157656864-f13b7d4a-f482-4b71-af62-06576c46f022.png)
 
-> Description the project.
 
 
 ## Built With
 
-- Major languages
-- Frameworks
-- Technologies used
+- Javascript, HTML, CSS
+- Bootstrap
+- weblint, eslint, hint
 
-## Live Demo (if available)
+## Live Demo
 
-[Live Demo Link](https://livedemo.com)
+[Live Demo Link](https://regiss05.github.io/game_ranking/)
 
```

Comment:
- [ ] The live demo link isn't currently working, please make sure to check that out.

Reply:
Addressed 🙌